openmc/polynomial.py

```diff
@@ -75,7 +75,68 @@ def order(self):
     def __call__(self, r):
         import openmc.lib as lib
         if isinstance(r, Iterable):
-            return [np.sum(self._norm_coef * lib.calc_zn_rad(self.order, r_i))
+            return [np.sum(self._norm_coef * lib.calc_zn_rad(self.order, r_i / self.radius))
                     for r_i in r]
         else:
-            return np.sum(self._norm_coef * lib.calc_zn_rad(self.order, r))
+            return np.sum(self._norm_coef * lib.calc_zn_rad(self.order, r / self.radius))
+
+
+class Zernike(Polynomial):
+    """Create Zernike polynomials given coefficients and domain.
+    The Zernike polynomials are defined as in
+    :class:`ZernikeFilter`.
```

Comment:
Combine this with previous line, and then add a blank line before and after it, i.e.
```
Create Zernike polynomials ...

The Zernike polynomials are defined as in :class:`ZernikeFilter`.

Parameters
...
```